app/models/concerns/exportable.rb

```diff
@@ -56,7 +56,14 @@ module Exportable
     "Notes Count" => :notes_count,
 
     # TODO clinic stuff
-    # TODO external pledges
+
+    # External Pledges
+    "External Pledge Count" => :external_pledge_count,
```

Comment:
Since there's no PII here, what are your thoughts about taking a slightly different tack and just raw-dumping everything into a single concat'd column? Something like

```
def external_pledges
  external_pledges.map { |x| "#{x.source} - #{x.amount}" }.join('; ')
end
```

and then parse the raw result in whatever py script? This kind of approach might get clunky for calls since we're dealing with quite a few potentially, but for cases like this where almost all will have 0-2 it might work pretty cleanly. Thoughts?

Reply:
I actually like that idea a lot in general, but how would you feel about having pledge count, pledge total, and then the raw result? It would make my life slightly easier

Reply:
I'm good with that

Reply:
Updated to reflect decision!